lib/rust/profiler/data/src/bin/devtools.rs

```diff
@@ -133,7 +133,7 @@ impl<'p, Metadata> IntervalTranslator<'p, Metadata> {
             self.events.push(event);
         }
         for child in &active.children {
-            self.visit_interval(*child, row + 1);
+            self.visit_interval(*child, _row + 1);
```

Comment:
why this change? `_row` seems to be used.

Reply:
Clippy complained because the variable is used only in recursion. Apparently, variables only used for recursion should have a leading underscore.

Reply:
Blergh. I think I missed what cluppy was trying to tell me: that variable is ONLY used in recursion. It seems not to do anything and can be removed.

Reply:
Removed.